portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,9 +34,9 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final String COMMENT_TABLE_NAME = "Comment";
-  private static final String COMMENT_COLUMN_NAME = "text";
-  private static final String TIMESTAMP_COLUMN_NAME = "submit_time";
+  public static final String COMMENT_TABLE_NAME = "Comment";
+  public static final String COMMENT_COLUMN_NAME = "text";
+  public static final String TIMESTAMP_COLUMN_NAME = "submit_time";
```

Comment:
nit - Since these servlets are all in the same package, you can use [package private visibility](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html). In general, prefer the least level of visibility possible.

Reply:
oooh cool I did not know about that! updated!